src/JustSaying/Fluent/AccountAddressProvider.cs

```diff
@@ -106,10 +106,12 @@ public Uri GetQueueUriByConvention<T>()
     /// <returns>The <see cref="Uri"/> for this queue.</returns>
     public Uri GetQueueUri(string queueName)
     {
+#pragma warning disable CS0618 // Type or member is obsolete
         var hostname = _regionEndpoint.GetEndpointForService("sqs").Hostname;
```

Comment:
This is obsolete, but using the new way is difficult to do for libraries without breaking compatibility or making things harder for consumers, so keeping using the old way.

Reply:
This will need revisiting with AWSSDK v4 as this has been removed completely.